sql/core/src/test/scala/org/apache/spark/sql/execution/datasources/parquet/ParquetIOSuite.scala

```diff
@@ -1070,17 +1070,6 @@ class ParquetIOSuite extends QueryTest with ParquetTest with SharedSparkSession
     }
   }
 
-  test("SPARK-35640: int as long should throw schema incompatible error") {
```

Comment:
also cc @sunchao 

Will this pr solve the problem described in SPARK-35461 too?

Reply:
Yes, this will address [SPARK-35461](https://issues.apache.org/jira/browse/SPARK-35461) - covered by this test case in particular: https://github.com/apache/spark/pull/44368/files#diff-a5cfd7285f9adf95b2aeea90aa57cc35d2b8c6bddaa0f4652172d30a264d3614R153